packages/devtools_app/lib/src/screens/logging/logging_screen.dart

```diff
@@ -122,13 +123,15 @@ class _LoggingScreenState extends State<LoggingScreenBody>
           onPressed: controller.clear,
           gaScreen: gac.logging,
           gaSelection: gac.clear,
+          minScreenWidthForTextBeforeScaling: loggingMinVerboseWidth,
         ),
         const Spacer(),
-        const StructuredErrorsToggle(),
         const SizedBox(width: denseSpacing),
         // TODO(kenz): fix focus issue when state is refreshed
         SearchField<LoggingController>(
-          searchFieldWidth: wideSearchFieldWidth,
+          searchFieldWidth: includeText(context, loggingMinVerboseWidth)
```

Comment:
maybe rename `includeText` to `screenWiderThanMin` or `useNarrowDisplay` or similar since this applies to more cases than just including text